src/pages/index.js

```diff
@@ -62,13 +134,53 @@ const cardSection = new Section(
   ".cards__list",
 );
 
-cardSection.renderItems();
-
 const editFormValidator = new FormValidator(config, profileEditForm);
 const addFormValidator = new FormValidator(config, addImageForm);
+const profileImageFormValidator = new FormValidator(config, profileImageForm);
+const profileEditFormValidator = new FormValidator(config, profileEditForm);
+
+profileImageFormValidator.enableValidation();
+
+function handleDeleteClick(cardToDelete) {
+  deleteConfirmationModal.open();
+
+  deleteConfirmationModal.setSubmitHandler(() => {
+    if (cardToDelete) {
+      api
+        .deleteCard(cardToDelete.id)
+        .then(() => {
+          cardToDelete.handleDeleteButton();
+          cardToDelete = null;
+          deleteConfirmationModal.close();
+        })
+        .catch((err) => console.error(err));
+    }
+  });
+}
+
+function handleLikeButton(card) {
+  if (card._isLiked) {
+    api
+      .unlikeCard(card.id)
+      .then((cardData) => {
+        card._isLiked = false;
+        card.updateLikes();
+      })
+      .catch((err) => console.error(err));
+  } else {
+    api
+      .likeCard(card.id)
+      .then(() => {
+        card._isLiked = true;
+        card.updateLikes();
+      })
+      .catch((err) => console.error(err));
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors